app/README.md

```diff
@@ -0,0 +1,32 @@
+# App
+
+Contains the source code for the backend piece of Civic Tech Jobs
+
+---
+## Overview
+TBD
+
+---
+## Getting Started
+### Pre-requisites
+- python 3.X
+  - pip
+  - venv
+- docker
+  - docker-compose
+
+### Setup Local Environment
+- Clone this repo
+- Navigate into this directory
+  - `cd ./CivicTechJobs/app`
+- Setup your python virtual environment and install dependencies
+  - `python -m venv .venv`
+  - `source .venv/bin/activate`
+  - `pip install -r ./requirements.txt`
+    - To deactivate virtual environment, type `deactivate`
```

Comment:
I am afraid that having a python venv would make having docker for dev a tad redundant (that said having `node_modules` outside of the container is also redundant). I would actually like for all dependencies to eventually move into the container, if possible.

Can the linter run via docker-compose in some way, such as through a dedicated linting container and the commit hook would run the docker-compose command before code commit?

Reply:
Thanks @Aveline-art  !
* deleted new `README` and moved contents to mkdocs section
* changed to node:18
* set new merge branch destination
* moved all linting dependencies to container that can now be invoked via `docker-compose` command. Have provided updated steps in `CivicTechJobs/mkdocs/docs/developer/installation.md`
  * this new approach uses the pre-commit.com framework but it does not create the commit hook quite yet. I think we can set this aside as a later TODO
  * also, hadolint (which is run in GH actions) is not yet incorporated into this container either. Have provided workaround in the doc, but I think we can set this also aside as a TODO as well